src/ModelTable/ModelTable.tsx

```diff
@@ -84,17 +84,19 @@ export const ModelTable = Object.assign(
                 typeof scrollable === 'object' ? scrollable?.className : ''
               }
             >
-              <Table columnIds={tableColumns} data={data} {...tableProps}>
-                {children === undefined ? (
-                  <>
-                    <ModelTableHeader />
-                    <ModelTableBody />
-                    <Table.Fallback />
-                  </>
-                ) : (
-                  children
-                )}
-              </Table>
+              <div className="relative h-full">
```

Comment:
What was the reason for having a div here?

Reply:
I found [this solution](https://github.com/shadcn-ui/ui/issues/1151#issuecomment-2309225952) which required wrapping it in a div with max-height, though it seems like I could've just used the existing ScrollAreaWrapper instead of adding another div. Moved the class over to that, and removed the wrapping div as well as the seemingly unnecessary relative class, thanks for the catch!